src/client/Microsoft.Identity.Client/Platforms/netcore/NetCoreHttpClientFactory.cs

```diff
@@ -10,6 +10,7 @@ namespace Microsoft.Identity.Client.Platforms.netcore
     internal class NetCoreHttpClientFactory : IMsalHttpClientFactory
     {
         private static HttpClient s_httpClient;
+        private static object s_lock = new object();
```

Comment:
consider changing this to leverage the lazy construction we do elsewhere in the product. this will remove the need for the lock and the null check